hrqb/base/target.py

```diff
@@ -21,11 +21,11 @@ def __init__(self, path: str, table_name: str) -> None:
 class PandasPickleTarget(HRQBLocalTarget):
     """Target is Pandas object (DataFrame or Series) pickled to disk."""
 
-    def read(self) -> PandasObject:
+    def read(self) -> pd.DataFrame:
```

Comment:
Minor comment: Since `PandasPickleTarget.read/write` has been updated to return instances only of type `pd.DataFrame`, can you update the class docstring as well?

Reply:
Good catch!  Thanks, will do.